protocol/messenger_group_chat.go

```diff
@@ -806,7 +799,7 @@ func (m *Messenger) DeclineAllPendingGroupInvitesFromUser(response *MessengerRes
 	}
 
 	// Decline group invites from activity center notifications
-	notifications, err := m.persistence.AcceptActivityCenterNotificationsForInvitesFromUser(userPublicKey, m.getCurrentTimeInMillis())
+	notifications, err := m.AcceptActivityCenterNotificationsForInvitesFromUser(ctx, userPublicKey, m.GetCurrentTimeInMillis())
```

Comment:
Same here, shouldn't we sync accept/decline result?

Reply:
Ok, @qfrank pointed me to that it's synced in `AcceptActivityCenterNotificationsForInvitesFromUser`. No idea how I missed that 🤦